app/Http/Controllers/PostReplyController.php

```diff
@@ -55,7 +55,7 @@ public function store(PostReplyRequest $request, Post $post, PostReply $postRepl
         if ($postReply) {
             $postReply->reply()->save($newPostReply);
 
-            return redirect()->back()->withStatus('Postreply successfully created.');
+            return redirect()->back()->withStatus('Comment successfully created.');
```

Comment:
@RheaMars @milack68 wenn wir jetzt von Comments und nicht von replies sprechen, sollten wir das Model entsprechend umbenennen. Dadurch werden die Beziehungen auch einfacher verständlich.

Reply:
@Cosnavel Wollte euch jetzt mit dem Test nicht in die bisherige Struktur pfuschen. Ich hatte es so verstanden:

- Ein Post ist das Element auf oberster Ebene, abgebildet durch das Post-Model.
- Eine PostReply ist die **direkte** Antwort auf einen Post, abgebildet durch das PostReply-Model.
- Ein Comment ist die Antwort auf eine PostReply, ebenfalls abgebildet durch das PostReply-Model.

Korrekt so?

Was mir noch ein Rätsel ist: Bislang (bis zu diesem PR) hatte die store-Methode im PostReplyController keinen Default-Null-Wert für $postReply. Dadurch sind wir **immer** in den Zweig `if ($postReply)`-case reingelaufen. Wieso hat das überhaupt funktioniert?